code/game/machinery/cryopod.dm

```diff
@@ -192,12 +192,13 @@
 		/obj/item/device/paicard,
 		/obj/item/weapon/gun,
 		/obj/item/weapon/pinpointer,
-		/obj/item/clothing/suit,
```

Comment:
Do we really need to record all of that? I would say for important items that need that info, we could store it elsewhere and reference it in the list so when the item is created, it adopts that information.